crates/secret-service-client/src/musig2.rs

```diff
@@ -91,6 +111,14 @@ pub struct Musig2FirstRound {
     config: Arc<Config>,
 }
 
+impl Musig2FirstRound {
+    /// Exports internal session id from the session struct. This leaks the internals and should be
+    /// used with caution.
+    pub fn dangerous_export_session_id(&self) -> Musig2SessionId {
+        self.session_id
+    }
```

Comment:
Don't we need this for the `MuSig2SecondRound` as well?

Reply:
Honestly I think we just ditch these methods altogether.